include/GafferUI/CompoundNodule.h

```diff
@@ -38,6 +38,7 @@
 #ifndef GAFFERUI_COMPOUNDNODULE_H
 #define GAFFERUI_COMPOUNDNODULE_H
 
+#include "Gaffer/Plug.h"
 #include "GafferUI/Nodule.h"
```

Comment:
foo

Reply:
Since you're here, I should mention that I'm finally in the process of getting this branch merged. I've dealt with all the conflicts, and I'm now just agonising over some details of the drag proxy and labelling stuff. Once I'm done I'll hand back to you for a final review/test.